CHAOS_MONKEY.md

```diff
@@ -0,0 +1,69 @@
+# Can we Chaos Monkey it?
+
+A goal for the project is to be able to tear down any of our build or test hosts 
+and recreate it completely from our Ansible playbooks.
+
+## Security and Patching
+
+Ansible must ensure that the underlying O/S is patched as well as any firewalls, 
+VPN and other security configured before making the host available.
+
+## Core Infrastructure
+
+Provider | Host | role | Chaos Monkey? | Issue(s) |
```

Comment:
I presume "issue" here is intended to be a link to an issue that would cover fixing any problem with the ansible scripts not regenerating everything as expected?

Reply:
Yes